src/fontra/client/web-components/add-remove-buttons.js

```diff
@@ -0,0 +1,46 @@
+import { html, css, LitElement } from "https://cdn.jsdelivr.net/npm/lit@2.6.1/+esm";
+
+export class addRemoveButtons extends LitElement {
+  static styles = css`
+    .buttons-container {
+      padding: 0.5em;
+    }
+
+    button {
+      min-width: 2em;
+    }
+
+    button:enabled:hover {
+      cursor: pointer;
+    }
+  `;
+
+  static properties = {
+    addButtonCallback: { type: Function },
+    removeButtonCallback: { type: Function },
+    disableRemoveButton: { type: Boolean },
+  };
+  constructor() {
+    super();
+    this.addButtonCallback = () => {};
+    this.removeButtonCallback = () => {};
+    this.disableRemoveButton = false;
+  }
+
+  render() {
+    return html`
+      <div class="buttons-container">
+        <button name="add-button" @click=${() => this.addButtonCallback()}>+</button>
```

Comment:
Can you show what Lit is adding to this Web Component apart from the `@click` syntax? Because if that's all, we might as well use `onclick=...`